### ISSUE TITLE
Regarding NTU-RGBD code

### ISSUE THREAD

**Issue by preksha12** [Maghoumi/DeepGRU#1]
Could you please provide code for NTU-RGBD dataset?

**Comment by Maghoumi** [comment]
What do you mean exactly? You mean loading the dataset?

**Comment by preksha12** [comment]
Yes  whole code if there is anything which is modified in case of NTU-RGBD dataset for CS and CV.

**Comment by preksha12** [comment]
Yes. Complete code for NTU-RGBD dataset

**Comment by preksha12** [comment]
I have tried to run code in google cola but when I am running main.py error is num_classes not defined.

**Comment by xl-huo** [comment]
Thank you for your fabulous work. Could you please provide the code for processing the NTU-RGBD data which contain multiple skeletons? If you could provide the dataset code, it's would be better.
Thanks.

**Comment by preksha12** [comment]
@rust-in Have you implemented DeepGRU for NTU-RGBD 60 dataset. Please provide code for the same.

**Comment by ancilcleetus** [comment]
@Maghoumi Right now, I'm creating a Hand Sanitization Detector for Hospitals. I have found your paper to be really interesting, especially good results with small training data, possibility of real time deployment for devices with low processing power.

I want to train DeepGRU on some custom classes in NTU Action Recognition Dataset. Can you please add any supporting material for loading NTU Dataset in your repo?

**Comment by FMMStudy** [comment]
Thank you for your fabulous work. Could you please provide the code for processing the NTU-RGBD data which contain multiple skeletons? If you could provide the dataset code, it's would be better.
Thanks.
